index.js

```diff
@@ -1097,7 +1097,7 @@ function maybeUnslab (block) {
 }
 
 function checkSnapshot (snapshot, index) {
-  if (index >= snapshot.state.snapshotCompatLength) throw SNAPSHOT_NOT_AVAILABLE()
+  if (index >= snapshot.state.snapshotCompatLength) throw SNAPSHOT_NOT_AVAILABLE(undefined, snapshot.discoveryKey)
```

Comment:
add index in the message

Reply:
Went for `snapshot at index ${index} not available (max compat length ${snapshot.state.snapshotCompatLength})`